modules/creating-a-project-workbench.adoc

```diff
@@ -34,7 +34,7 @@ The *Create workbench* page opens.
 .. In the *Deployment size* section, specify the size of your deployment instance.
 ... From the *Container size* list, select a container size for your server.
 ... Optional: From the *Accelerator* list, select an accelerator. 
-... Optional: If you selected an accelerator in the preceding step, from the *Number of accelerators* list, select a value.  
+.... From the *Number of accelerators* list, select a value.  
```

Comment:
I think we still want the conditional statement, but can probably make the second half simpler also:

"If you selected an accelerator in the preceding step, select the number of accelerators to use."

Reply:
I'd nested the step under the first optional one - can revert that and add the conditional statement back in if it aligns better with style guidelines/what we already have?

Reply:
Yeah, I'd move it back to being a standalone step.